docs/user-guide_installation.md

```diff
@@ -71,20 +79,21 @@ Select the following tabs for the installation description of your target.
     ![DietPi-Etcher-install-03](assets/images/DietPi-Etcher-install-03.jpg)
 
     ??? info "Click here if you want to pre-configure WiFi network "
-    	To setup Wifi, open the SD card folder, and update next two files using a text editor of your choice:
+      	To setup Wifi, open the SD card folder, and update next two files using a text editor of your choice:
```

Comment:
A bid inconsistent indentation here. Two spaces should be sufficient for content or warnings and expandable boxes, but four characters might look a bid cleaner in Markdown code as content then exactly aligns below header? However, currently sometimes we use 2, sometimes 3, sometimes 4 spaces 😉.

Reply:
Jep, hit!  
I had some problems exactly at these points in the text (infos and warnings). With a larger indentation it worked better in the `mkdocs serve` displaying.    
Why I don't know. I also assumed that 2 spaces are sufficient...

I will have a look on this playing a bit with tabs and spaces. Possibly only the usage of tabs was the reason.